support-frontend/assets/pages/digital-subscription-landing/components/productBlock.jsx

```diff
@@ -115,15 +110,22 @@ const appImage = (
 
 type ProductCardPropTypes = {
   title: string,
-  subtitle: Node,
+  subtitle: string,
+  shortSubTitle: string | null,
   image: Node,
+  second: boolean,
 }
 
-const ProductCard = ({ title, subtitle, image }: ProductCardPropTypes) => (
+const ProductCard = ({
+  title, subtitle, shortSubTitle, image, second = false,
+}: ProductCardPropTypes) => (
   <section className="product-block__item">
     <h2 className="product-block__item__title">{title}</h2>
-    <p className="product-block__item__subtitle">{subtitle}</p>
-    <span className="product-block__item__image">{image}</span>
+    <p className="product-block__item__subtitle">
+      <span className={`product-block__item__subtitle--desktop${second ? '--second' : ''}`}>{subtitle}</span>
+      {!second && <span className="product-block__item__subtitle--mobile">{shortSubTitle}</span>}
```

Comment:
If you makee `shortSubTitle` optional you could update this to, so we don't render an empty `<span>` in the event no sub title exists.

```jsx
{!second && shortSubTitle && <span className="product-block__item__subtitle--mobile">{shortSubTitle}</span>}
```

Reply:
👍  Thanks George